### ISSUE TITLE
The derep_fulllength command fails to remove the part of the header after the space

### ISSUE THREAD

**Issue by torognes** [torognes/vsearch#338]
The derep_fulllength command fails to remove the part of the header after the space.

The part of the header line from the first space should be ignored (unless the --notrunclabels option is in effect). This does not seem to apply to the derep_fulllength command, but it should.

**Comment by frederic-mahe** [comment]
```sh
vsearch --derep_fulllength <(printf ">header metadata\nA\n") --output - --minseqlength 1 --sizeout --quiet
```
gives:
```
>header metadata;size=1
A
```
instead of:
```
>header;size=1
A
```
I've written about a hundred tests for the dereplication-rereplication functions, but I have to admit I did not test the header parsing.

**Comment by frederic-mahe** [comment]
I've added two new tests for that specific issue (https://github.com/frederic-mahe/vsearch-tests/commit/620cd91bfe716453eb0ae3e4575673849559b91d).

**Comment by torognes** [comment]
The bug was introduced in version 2.8.2, but is fixed in version 2.8.6 just released.
